src/main/java/javaserver/RequestBuilder.java

```diff
@@ -3,23 +3,20 @@
 import java.util.HashMap;
 
 public class RequestBuilder {
-	
-	public HashMap<String, String> buildRequest(String rawRequest) {
+
+	public HashMap<String, String> getRequestObject(String rawRequest) {
```

Comment:
It might make sense to create a `Request` object instead of using a HashMap. In that way, you can expose things like the type, URI, and data with getters and the client using `RequestBuilder` doesn't need to know the internals of the map to retrieve what they need.